.circleci/config.yml

```diff
@@ -95,15 +95,25 @@ commands:
       - run:
           working_directory: Tests/CodegenCLITests/pod-install-test/
           command: pod install --verbose
-          name: Install Pod (verbose)
+          name: CocoaPods - Install
       - run:
           working_directory: Tests/CodegenCLITests/pod-install-test/Pods/Apollo/
           command: ./apollo-ios-cli init --schema-name NewTestSchema
-          name: Test init
+          name: CocoaPods - CLI Test (init)
       - run:
           working_directory: Tests/CodegenCLITests/pod-install-test/Pods/Apollo/
           command: ./apollo-ios-cli generate
-          name: Test generate
+          name: CocoaPods - CLI Test (generate)
+  swiftpm_plugin_test:
+    steps:
+      - run:
+          working_directory: Tests/CodegenCLITests/swiftpm-test/
+          command: swift build
+          name: SwiftPM - Build package
+      - run:
+          working_directory: Tests/CodegenCLITests/swiftpm-test/
+          command: swift package plugin --allow-writing-to-package-directory apollo-generate --verbose
```

Comment:
We only test the `generate` command here because we needed have an existing JSON config file to prevent codegen finding all the test graphQL definition files in the `.build` folder. We're going to fix that behaviour in #2552.